### PR TITLE
POST purge link, to avoid confirmation

### PR DESCRIPTION
**Pull request by samwilson**
This takes the code from https://www.mediawiki.org/w/index.php?title=MediaWiki:Gadget-UTCLiveClock.js to make purging a one-click action.

### CODE REVIEW

**Comment by Hutchy68** [comment]
Thanks, it's been awhile since I even looked at this extension although I use it. Will have to prep a new release.

**Comment by samwilson** [comment]
Thanks!

File: resources/ext.purge.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+mw.loader.using( [ 'mediawiki.api', 'mediawiki.notify' ] ).then( function () {
+
+	$( "#ca-purge a" ).on( 'click', function ( e ) {
+		var postArgs = { action: 'purgee', titles: mw.config.get( 'wgPageName' ) };
```
Comment (kghbln) [inline resources/ext.purge.js]: @samwilson This should be purge instead of purgee. Spotted by @mwjames

Comment (samwilson) [inline resources/ext.purge.js]: Thanks!! I feel silly. 

Fixed in #9.

Comment (kghbln) [inline resources/ext.purge.js]: No worries. A fix is always just a pull request away.

**Pull request merged**